docs/spark/how-to-guides/debug.md

```diff
@@ -6,9 +6,9 @@ ms.topic: conceptual
 ms.custom: mvc,how-to
 ---
 
-# Debug a .NET for Apache Spark application
+# Debugging .NET for Apache Spark Application
```

Comment:
This change should be reverted, I think.
The guidelines is to avoid "ing" and to use sentence-casing.
So, it should be "Debug" and "application" instead of "Debugging" and "Application"

Reply:
Ah ok sure will do. Thanks!